src/Maestro/Maestro.Data/Migrations/BuildAssetRegistryContextModelSnapshot.cs

```diff
@@ -607,12 +607,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.HasOne("Maestro.Data.Models.Channel", "Channel")
                         .WithMany("ChannelReleasePipelines")
                         .HasForeignKey("ChannelId")
-                        .OnDelete(DeleteBehavior.Restrict);
+                        .OnDelete(DeleteBehavior.Cascade);
 
                     b.HasOne("Maestro.Data.Models.ReleasePipeline", "ReleasePipeline")
                         .WithMany("ChannelReleasePipelines")
                         .HasForeignKey("ReleasePipelineId")
-                        .OnDelete(DeleteBehavior.Restrict);
+                        .OnDelete(DeleteBehavior.Cascade);
```

Comment:
It looks like Cascade again... talk with @alexperovich

Reply:
I think I got it right this time :-)